core/input/input_event.cpp

```diff
@@ -762,7 +834,40 @@ bool InputEventJoypadMotion::action_match(const Ref<InputEvent> &p_event, bool *
 	return match;
 }
 
+// Maps to 2*axis if value is neg, or + 1 if value is pos
+static const char *_joy_axis_descriptions[JOY_AXIS_MAX * 2] = {
+	TTRC("Left Stick Left"),
+	TTRC("Left Stick Right"),
+	TTRC("Left Stick Up"),
+	TTRC("Left Stick Down"),
+	TTRC("Right Stick Left"),
+	TTRC("Right Stick Right"),
+	TTRC("Right Stick Up"),
+	TTRC("Right Stick Down"),
+	TTRC("Joystick 2 Left"),
+	TTRC("Joystick 2 Right, Left Trigger, L2, LT"),
+	TTRC("Joystick 2 Up"),
+	TTRC("Joystick 2 Down, Right Trigger, R2, RT"),
+	TTRC("Joystick 3 Left"),
+	TTRC("Joystick 3 Right"),
+	TTRC("Joystick 3 Up"),
+	TTRC("Joystick 3 Down"),
+	TTRC("Joystick 4 Left"),
+	TTRC("Joystick 4 Right"),
+	TTRC("Joystick 4 Up"),
+	TTRC("Joystick 4 Down"),
+};
+
```

Comment:
These shouldn't be defined twice:
https://github.com/godotengine/godot/blob/c9bd11a479099bf9d0e9d3cb61f9bf362e76344e/editor/input_map_editor.cpp#L56-L77
There should be consistency between the Editor Shortcut descriptions and the Input Mapping descriptions. And, it probably makes sense to bring the descriptions for the Input Mapping here as part of this PR.
However, as [mentioned](https://github.com/godotengine/godot/pull/43660#issuecomment-730221162) by @akien-mga, these are in the process of being updated in #43591; so let's make sure they make sense in both contexts.